es

```diff
@@ -42,8 +42,8 @@ PHPMD_RULESETS=${PHPMD_RULESETS:=cleancode,codesize,controversial,design,naming,
 ########## PHPSTAN CONFIGURATION ##########
 # Whether or not to run phpstan, will run if phpstan binary is found
 PHPSTAN_ENABLED=${PHPSTAN_ENABLED:=true}
-# The reporting level, 1 = loose, 7 = strict
-PHPSTAN_REPORTING_LEVEL=${PHPSTAN_REPORTING_LEVEL:=7}
+# The reporting level, 1 = loose, 8 = strict
+PHPSTAN_REPORTING_LEVEL=${PHPSTAN_REPORTING_LEVEL:=8}
```

Comment:
This will fail unless they're using 0.12, you'll need to check the exe path first, then set the level like bamboo